ui/src/components/Input/Input.tsx

```diff
@@ -26,7 +26,7 @@ export default function Input({
     <div className={style.inputContainer}>
       {label && <label>{label}</label>}
       <div className={style.input}>
-        <input name={name} type={type} value={value} onChange={(e) => onChange(e)} placeholder={placeholder} />
+        <input name={name} type={type} value={value || ''} onChange={(e) => onChange(e)} placeholder={placeholder} />
```

Comment:
It's not necessary to dynamically assign a function in props onChange. OnChange can be assgined to the props directly:
```
onChange={onChange}
```

Reply:
fix in 569812e